pyproject.toml

```
@@ -13,17 +13,17 @@ classifiers = ["Topic :: Software Development :: Libraries :: Python Modules"]
 
 [tool.poetry.dependencies]
 # The `<4` restriction is because of coverage.py.
-python = ">=3.8,<4"
+python = ">=3.5.9,<4"
```

```suggestion
python = ">=3.6,<4"
```
And if we were keeping 3.5 support you shouldn't restrict the bugfix version.

---

Yes, thanks for the correction. But let's drop 3.5 altogether!